app/src/main/java/com/example/event_finder_capstone/MainActivity.java

```diff
@@ -1,14 +1,150 @@
+
+        getAPIEvents();
+    }
+
+    private void getAPIEvents() {
+        String eventSearchRegion = "en_US";
+        Long eventSearchRadiusFromUser = 40000L;
+        String numberOfEventsToRetrieve = "10";
+        Long upcomingEventsOnly = (System.currentTimeMillis() / 1000L);
+        RetrofitClient.getInstance().getYelpAPI().getEvents(eventSearchRegion,
+                numberOfEventsToRetrieve,
+                upcomingEventsOnly,
+                eventSearchRadiusFromUser,
+                ParseUser.getCurrentUser().getString("zip")).enqueue(new Callback<JsonObject>() {
+            @Override
+            public void onResponse(@NonNull Call<JsonObject> call, @NonNull Response<JsonObject> response) {
+                if (response.isSuccessful() && response.body() != null) {
+                    runOnUiThread(() -> {
+                        try {
+                            JsonObject result = response.body();
+                            eventsList.clear();
+                            eventsList.addAll(convertToList(result));
+                            eventsAdapter.notifyDataSetChanged();
+                        } catch (Exception e) {
+                            Toast.makeText(MainActivity.this, "JSON exception error", Toast.LENGTH_SHORT).show();
+                        }
+                    });
+                } else {
+                    Toast.makeText(MainActivity.this, "Query Failed", Toast.LENGTH_SHORT).show();
+                }
+            }
+
+            @Override
+            public void onFailure(@NonNull Call<JsonObject> call, @NonNull Throwable t) {
+                Toast.makeText(MainActivity.this, "Failed to call API", Toast.LENGTH_SHORT).show();
+            }
+        });
+    }
+
+    private Collection<? extends Event> convertToList(JsonObject result) {
+        List<Event> res = new ArrayList<>();
+        JsonArray events = result.getAsJsonArray("events");
+        for (int i = 0; i < events.size(); i++) {
+            JsonObject temp = (JsonObject) events.get(i);
+            Event event = new Event();
+            populateEventInfo(i, event, temp);
+            res.add(event);
+        }
+        return res;
+    }
+
+    private void populateEventInfo(int  i, Event event, JsonObject temp) {
+        event.setName(temp.get("name").getAsString());
+        event.setDescription(temp.get("description").getAsString());
+        event.setImageUrl(temp.get("image_url").getAsString());
+        event.setTimeStart(temp.get("time_start").getAsString());
+        event.setEventSiteUrl(temp.get("event_site_url").getAsString());
+        checkAndSetEventEndTime(event, temp);
+        checkAndSetEventCost(event, temp);
+        formatAndSetEventLocation(temp, event);
+    }
+
+    private void checkAndSetEventCost(Event event, JsonObject temp) {
+        if (!String.valueOf(temp.get("cost")).equals("null")) {
+            event.setCost("$" + temp.get("cost").getAsString() +"0");
```

Comment:
⚠️ Note: Make sure you're consistently formatting code (as well as any warnings) as you go along.  I find a checklist can be useful for consistent items I want to do before raising a PR.